extensions/wikia/CreatePage/js/CreatePage.js

```diff
@@ -20,15 +21,18 @@ var CreatePage = {
 		},
 		function( response ) {
 			var articlePath;
+			var flowParam = CreatePage.flowName === '' ? '' : '&flow=' + CreatePage.flowName;
```

Comment:
Please use parentheses - assigning the value of trinary expression reads slower :)